src/core/libmaven/mzUtils.cpp

```diff
@@ -7,274 +7,215 @@
 #include <boost/iostreams/copy.hpp>
 #include <boost/iostreams/filter/zlib.hpp>
 #include <boost/iostreams/filtering_streambuf.hpp>
+#include <boost/algorithm/string.hpp>
 
 /**
  * random collection of useful functions 
  */
 namespace mzUtils {
 
-    int randInt(int low, int high){
+    const long double   Pi = 3.1415926535897932384626433832795028841968;
```

Comment:
Do we not get a `Pi` constant from the standard math library? We get `M_PI` from the cmath library, that's for sure.